contributing/getting-started-e2e.md

```diff
@@ -396,16 +396,16 @@ The `amphtml` testing framework uses the [Mocha](https://mochajs.org/), [Chai](h
 
 Existing components will usually have existing tests that you can follow as an example. For example, the [amp-video](../extensions/amp-video/amp-video.md) component has tests in [test/test-amp-video.js](../extensions/amp-video/0.1/test/test-amp-video.js).
 
-To run tests in a single file, use `gulp test --files=<path>`:
+To run tests in a single file, use `gulp unit --files=<path>`:
 
 ```
-gulp test --files=extensions/amp-youtube/0.1/test/test-amp-youtube.js
+gulp unit --files=extensions/amp-youtube/0.1/test/test-amp-youtube.js
```

Comment:
Add an example of how to run two files? (it might not clear to new devs whether it's `--files=file1,file2` or `--files=file1 --files=file2`)